cobalt/dom/source_buffer_metrics.cc

```diff
@@ -71,8 +71,7 @@ void SourceBufferMetrics::StartTracking() {
   DCHECK(!is_tracking_);
   is_tracking_ = true;
   wall_start_time_ = starboard::CurrentMonotonicTime();
-  thread_start_time_ =
-      SbTimeIsTimeThreadNowSupported() ? SbTimeGetMonotonicThreadNow() : -1;
+  thread_start_time_ = starboard::CurrentMonotonicThreadTime();
```

Comment:
Doesn't this change the current behavior? If it wasn't supported before, it would return -1, vs now it will return 0. Is thread_start_time_ only used in the below calculation so it doesn't matter?